sdk/resourcemanager/containerservice/agent_pool/README.md

```diff
@@ -5,7 +5,7 @@ languages:
 products:
 - azure
 description: "These code samples will show you how to manage Container Service using Azure SDK for Golang."
-urlFragment: containerservice
+urlFragment: container-service
```

Comment:
Does this need the `agent-pool` suffix?

Reply:
You can still have "container-service" if you want. For the URL fragment, it just needs to be unique. Since no other samples are using the URL fragment "container-service", this should be okay. 

It could also be "container-service-agent-pool". It just cannot be "container-service-maintenance" or "container-service-managed-clusters". Up to your preference.

Reply:
Mostly I say this because I think it would be good if the URLs for the samples and the samples repo as are close as possible.

Reply:
Makes sense to me. I have updated the commit to be container-service-agent-pool